client/src/components/GraphViewerComponent/GraphViewerComponent.js

```diff
@@ -6,7 +6,7 @@ import { Icon } from "office-ui-fabric-react";
 
 import GraphViewerCommandBarComponent from "./GraphViewerCommandBarComponent/GraphViewerCommandBarComponent";
 import { GraphViewerCytoscapeComponent, GraphViewerCytoscapeLayouts } from "./GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent";
-import { GraphViewerRelationshipCreateComponent } from "./GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent";
+import GraphViewerRelationshipCreateComponent from "./GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent";
```

Comment:
Why did this change?

Reply:
The way the component is exported when we add the i18n library and the translator logic require the component to be exported different.